### PR TITLE
Fix/ruby documentation

### PR DESCRIPTION
**Pull request by sgrshah**
- Fixes Ruby Docs
- Adds Customer#delete
- Adds site requirement to Collection#create

### CODE REVIEW

File: source/includes/_resource.collections.md [hunk 1]
```diff
@@ -69,13 +69,15 @@ vhx.collections.create();
 $ curl -X POST "https://api.vhx.tv/collections" \
   -d name="Collection Name" \
   -d type="series" \
+  -d site="http://api.vhx.tv/sites/:id"
```
Comment (ksheurs) [inline source/includes/_resource.collections.md]: i'm trying to rid all references to `site`. can you use `product` as an example context here?

File: source/includes/_resource.collections.md [hunk 2]
```diff
@@ -69,13 +69,15 @@ vhx.collections.create();
...
   -u o3g_4jLU-rxHpc9rsoh3DHfpsq1L6oyM:
 ```
 
 ```ruby
 collection = Vhx::Collection.create({
   name: 'Collection Name',
-  type: 'series'
+  type: 'series',
+  site: 'http://api.vhx.tv/sites/:id'
```
Comment (ksheurs) [inline source/includes/_resource.collections.md]: same here

File: source/includes/_resource.collections.md [hunk 3]
```diff
@@ -158,6 +160,14 @@ vhx.collections.create({
       <td>The name of the collection.</td>
     </tr>
     <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">site</strong>
```
Comment (ksheurs) [inline source/includes/_resource.collections.md]: same here

File: source/includes/_resource.collections.md [hunk 4]
```diff
@@ -547,7 +557,7 @@ GET /collections/:id/items
 ```
 
 ```ruby
-Vhx::CollectionItem.list()
+Vhx::Item.all()
```
Comment (ksheurs) [inline source/includes/_resource.collections.md]: are other client libraries referencing item as a root level resource? they are only represented as a sub resource to collections. thoughts?

File: source/includes/_resource.videos.md [hunk 5]
```diff
@@ -433,7 +433,8 @@ $ curl -X GET "https://api.vhx.tv/videos/:id/files?quality=adaptive&format=m3u8"
 ```
 
 ```ruby
-files = Vhx::Video.files(465, {
+files = Vhx::File.all({
```
Comment (ksheurs) [inline source/includes/_resource.videos.md]: Thoughts on `VideoFile`? Similar feedback to Item vs CollectionItem.

File: source/includes/_resource.collections.md (hunk 4 above)
Comment (sgrshah) [inline source/includes/_resource.collections.md]: Node is representing like this: 

```
vhx.collections.items({
  collection: 'https://api.vhx.tv/collections/1',
  page: 1
});
```

We could represent in ruby as `Vhx::Collection::Item` or also only be able to get items like this `collection.items`

Any preferences?

File: source/includes/_resource.collections.md (hunk 1 above)
Comment (sgrshah) [inline source/includes/_resource.collections.md]: cool :+1:

File: source/includes/_resource.collections.md (hunk 4 above)
Comment (sgrshah) [inline source/includes/_resource.collections.md]: Could also do the same thing for `video.files`

Comment (ksheurs) [inline source/includes/_resource.collections.md]: interesting. hmmmmmmmmmm

let's stick with collection.items and video.files for now? only expose those larger objects later if need be?

**Comment by sgrshah** [comment]
cc @scottusrobus as well. Does this deploy automatically when i merge into master?

**Comment by scottusrobus** [comment]
@sgrshah it doesn't automatically deploy. I can do that for you, and/or show you how! ping me tomorrow for a pair if you wish.

**Pull request merged**